File: Application/Services/Operations/Finances/Dtos/VariableDebitExpenses/VariableExpenseDto.cs
using Application.Services.Operations.Finances.Dtos.InheritanceDto;

namespace Application.Services.Operations.Finances.Dtos.VariableDebitExpenses
{
    public class VariableExpenseDto : BaseExpenseDto
    {
        // public string Item { get; set; }
        public string Place { get; set; }

    }
}

File: Views/src/components/financial/components/variable-expenses/dto/variable-expense-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { BankAccountDto } from "../../bank-account-cards/dto/bank-account-dto";
import { CardDto } from "../../bank-account-cards/dto/card-dto";
import { PixDto } from "../../bank-account-cards/dto/pix-dto";
import { CategoryExpenseDto } from "src/components/financial/components/common-components/category-subcategory-expenses/dto/category-expense-dto"
import { SubcategoryExpenseDto } from "src/components/financial/components/common-components/category-subcategory-expenses/dto/subcategory-expense-dto"

import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { BaseExpenseDto } from "../../common-components/dto/base-expense-dto";
export class VariableExpenseDto extends BaseExpenseDto {
  item: string;
  place: string;
}
